http_calls.py

```diff
@@ -428,7 +434,10 @@ def __init__(self):
         global lastRequestResult
 
         o = json.loads(lastRequestResult)
-        self.result = o['hits']['hits']
+        if(type(o) == dict and "hits" in o):
```

Comment:
Test: ObjectEntity/GetMultipleInstances
Request: /api/objects/multiple returns items in 'doc' key but not in 'hits':
b'{"docs":[{..}]'